sdk/core/Azure.Core.Testing/src/Azure.Core.Testing.csproj

```diff
@@ -0,0 +1,27 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <Description>A set of testing primitives for libraries that use Azure.Core</Description>
+    <AssemblyTitle>Microsoft Azure Client Pipeline Testing</AssemblyTitle>
+    <Version>1.0.0-preview.5</Version>
+    <PackageTags>Microsoft Azure Client Pipeline Testing</PackageTags>
+
+    <!-- Make sure that we don't pull in additional dependencies during build or package -->
+    <ImportDefaultReferences>false</ImportDefaultReferences>
+
+    <!-- This is a workaorund until https://github.com/Azure/azure-sdk-for-net/issues/5214 is addressed -->
+    <RequiredTargetFrameworks>net461;netstandard2.0</RequiredTargetFrameworks>
```

Comment:
Do we still need this? Looks like the issue you are referencing was addressed/closed

Reply:
This entire file was removed in the followup PR https://github.com/Azure/azure-sdk-for-net/pull/6132